src/modules/process/ui/screens/shipment/redesign/process-detail-redesign.css

```
@@ -0,0 +1,834 @@
+  background: linear-gradient(
+    90deg,
+    transparent 0%,
+    var(--castro-gold) 40%,
+    var(--castro-gold-2) 60%,
+    transparent 100%
+  );
+  background-size: 200% 100%;
+  animation: castroDetailLoadingSweep 1.4s ease-in-out infinite;
+}
+
+@keyframes castroDetailLoadingSweep {
```

Stylelint (17.11.0)

[error] Expected keyframe name "castroDetailLoadingSweep" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)